contract/create_asset_allocation_request.go

```diff
@@ -1,14 +1,22 @@
 package contract
 
-import (
-	"github.com/google/uuid"
-)
+import "github.com/google/uuid"
+
+type CreateAssetAllocationRequestForJson struct {
```

Comment:
for what purpose this struct is used?

Reply:
Because the request body had only user_id and we had to send asset_id, admin_id and user_id to service and repo layer.